bip-emilengler-replacing-the-term-address.mediawiki

```diff
@@ -0,0 +1,46 @@
+<pre>
+  BIP: UNDEFINED
+  Titel: Replacing the term address
+  Author: Emil Engler <me@emilengler.com>
+  Status: Draft
+  Type: Informational
+  Created: 2019-10-17
+  License: CC0
+</pre>
+
+==Abstract==
+This BIP proposes a new term for 'address'
+
+==Specification==
+The new term is:
+''Bitcoin'' '''Invoice''' ''Address''
+
+The term ''Bitcoin'' and ''Address'' are optional this has the following reasons:
+For ''Bitcoin'':
+* Useful for multicoin wallets to indicate that it belongs to Bitcoin
+* Indicates a difference between a lightning and an on-chain invoice
+For ''Address'':
+* To not confuse users with a completely new term
+* To show that it is where you send something to
+
+This gives us the four following possibilities:
+* Bitcoin Invoice Address
+* Bitcoin Invoice
+* Invoice Address
+* Invoice
+
+==Motivation==
+Bitcoin addresses are intended to be only used '''once''' and you should generate a new one for every new incoming payment.
+The term 'address' however indicates consistency because nearly everything on the internet or the offline world with the term 'address'
+is something that rarely or even never changes (postal address, e-mail address, IP-Addresses (depends heavily on the provider), etc.)
+The motivation for this BIP is to change the term address to something that indicates that the address is connected to a single transaction.
+
+==Rationale==
+The reason why we use ''Bitcoin Invoice Address'' or just ''Invoice'' is to show that it is more like a temporary thing.
+We decided to keep Bitcoin and Address as optional terms for the reasons named above.
+
```

Comment:
A backwards compatibility section is needed.

Reply:
It was suggested to remove it 3fe8174e084a994bd6f8d4b4fdc474a36752b5d0
Was the old one sufficient?

Reply:
No, it doesn't actually address backward compatibility.

You want something like "since the term used will initially be `invoice address`, it includes the existing term `address` and should be immediately recognisable to users"